sirepo/template/controls.py

```diff
@@ -67,8 +76,10 @@ def sim_frame(frame_args):
 
 def _extract_report_elementAnimation(frame_args, run_dir, filename):
     data = frame_args.sim_in
-    if sirepo.template.madx.is_parameter_report_file(filename):
-        return sirepo.template.madx.extract_parameter_report(data, run_dir, filename)
+    if 'twiss' in frame_args.frameReport.lower():
```

Comment:
I think there is only one twiss report so you can be explicit here:
if frame_args.frameReport == 'instrumentAnimationTwiss':
    ...